src/controllers/organizations.js

```diff
@@ -1,5 +1,34 @@
-const { default: mongoose } = require('mongoose');
-const { Organization, User } = require('../models/user');
+const mongoose = require('mongoose');
+const { User, Organization } = require('../models/user');
+
+const addOrganizationSubscription = async (req, res) => {
+    try {
+        const followOrganizationId = mongoose.Types.ObjectId(req.params.id);
+
+        const organization = await Organization.findById(followOrganizationId);
+
+        if (!organization) {
+            res.status(404).json({ message: 'Organization not found' });
+        }
+
+        await User.findByIdAndUpdate(req.user.id, {
+            $push: {
```

Comment:
The user might be already following the organization, 
So instead of using $push, I suggest that you use $addToSet which will ensure that the `followedOrganizations` is always a set (an array with no duplicate values).

Reply:
never thought that! thanks !